ar_exercise/exercise.rb

```diff
@@ -31,37 +31,107 @@ class Apartment < ActiveRecord::Base
 
 # Use Active record to do the following, and store the results **in a variable**
 # example: get every tenant in the DB
+
 all_tenants = Tenant.all
 
+# puts all_tenants
+#
 # get the first tenant in the DB
+#
+firstTenant = Tenant.first
+
 # get all tenants older than 65
+#
+seniors = Tenant.where("age > 65", :age)
```

Comment:
This doesn't require the second argument because there's no value being subbed into the string. This is 'more correct':
`seniors = Tenant.where("age > 65")`